sdk/rust-language-support/examples/number-stream-accumulation/Cargo.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+name = "number-stream-accumulation"
+version = "0.1.0"
+authors = ["Shale Xiong <shale.xiong@arm.com>"]
```

Comment:
"The Veracruz Development Team".  Also, similar comment to above regarding a missing `description` field.